dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/Note.java

```diff
@@ -58,5 +58,8 @@
     private String value;
 
     @JsonProperty
-    private UserInfo createdBy;
+    private User updatedBy;
```

Comment:
Can a note be updated? I thought its immutable. It seems confusing to have an updatedBy field

Reply:
Done